include/highfive/bits/h5p_wrapper.hpp

```diff
@@ -239,6 +239,21 @@ inline herr_t h5p_set_szip(hid_t plist_id, unsigned options_mask, unsigned pixel
     return err;
 }
 
+#include "blosc_filter.h"
+
+inline herr_t h5p_set_blosc(hid_t plist_id, const unsigned int *cd_values) {
+    char *version, *date;
```

Comment:
These are both leaked, because `register_blosc` uses `strdup` which requires us to call `free`.